### PR TITLE
Fix typo in doctrine doc

### PR DESCRIPTION
**Pull request by Th3Mouk**

### CODE REVIEW

File: doctrine/event_listeners_subscribers.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@
 How to Register Event Listeners and Subscribers
 ===============================================
 
-Doctrine packages a rich event system that fires events when almost anything
+Doctrine packages had a rich event system that fires events when almost anything
 happens inside the system. For you, this means that you can create arbitrary
```
Comment (javiereguiluz) [inline doctrine/event_listeners_subscribers.rst]: `have` ?

Comment (Th3Mouk) [inline doctrine/event_listeners_subscribers.rst]: obviously too fast, i correct it

**Comment by xabbuh** [comment]
:+1:

Status: Reviewed

**Review by javiereguiluz — APPROVED** [review]
👍

File: doctrine/event_listeners_subscribers.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@
 How to Register Event Listeners and Subscribers
 ===============================================
 
-Doctrine packages a rich event system that fires events when almost anything
+Doctrine packages have a rich event system that fires events when almost anything
```
Comment (xabbuh) [inline doctrine/event_listeners_subscribers.rst]: This is just a test. Please ignore the comment.

**Review by xabbuh — CHANGES_REQUESTED** [review]
please ignore this review

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Good catch, thanks @Th3Mouk.

**Pull request closed without merging**